pkg/reconciler/certificate/config/cert_manager_test.go

```diff
@@ -62,19 +62,26 @@ func TestIssuerRef(t *testing.T) {
 		name:    "valid IssuerRef",
```

Comment:
Can we add a test case here for invalid IssuerKind?

Reply:
Thank you. I was planning to add initialization and validation later, but yes this validation might be better to add now.

Reply:
Ah cool. Thanks @nak3 !